hadoop-ozone/dist/src/main/compose/ozonesecure-mr/docker-compose.yaml

```diff
@@ -23,17 +23,23 @@ services:
       args:
         buildno: 1
     hostname: kdc
+    networks:
+      - ozone
```

Comment:
Does default network work for this case? why do we explicitly change the network name?

Reply:
Default network does not work, since it's name is `ozonesecure-mr_default`, which triggers `URISyntaxException` due to `_`.  Adding an explicit name avoids the `_default` suffix.